Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -23,12 +19,16 @@
 public final class GameEngine {
 
     private final World world;
+
+    private final Session session;
+    private final Room room;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "room" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)